xml/ha_config_basics.xml

```diff
@@ -657,9 +651,11 @@
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term>Stateful Clones (Multi-state Resources)</term>
+      <term>Promotable Clones (Multi-state Resources)</term>
       <listitem>
        <para>
+        <remark>taroth 2018-09-03: ygao, is the following still correct? if not,
+        how to adjust the following text? please advise!</remark>
```

Comment:
The text is still a correct description.

Reply:
Thanks, moved to XML comment for the records.